wayland-client/src/lib.rs

```diff
@@ -42,10 +42,13 @@
 //! managing the newly created object.
 //!
 //! However, implementing all those traits on your own is a lot of (often uninteresting) work. To make this
-//! easier a composition mechanism is provided using the [`delegate_dispatch!`] macro. This way, another
-//! library (such as Smithay's Client Toolkit) can provide generic [`Dispatch`] implementations that you
+//! easier a composition mechanism is provided. Using the [`QueueHandle<State>::make_data()`] method
+//! one is able to create object handlers manually, making it possible to specify `DispatchTo` type diferent
+//! then your `State`.
+//! Library (such as Smithay's Client Toolkit) can use that mechanism to provide generic [`Dispatch`] implementations that you
 //! can reuse in your own app by delegating those objects to that provided implementation. See the
-//! documentation of those traits and macro for details.
+//! documentation of [`QueueHandle<State>::make_data()`] for details.
+//! There is also `delegated.rs` example in `wayland-rs/wayland-client`.
```

Comment:
Maybe lets make this a  direct link to where the example file will be once this PR is merged?

Reply:
Done https://github.com/Smithay/wayland-rs/pull/699/commits/ee9140d38bbf6922653196a3fbe1200b175808cc#diff-82d250d3dc7f4360cf4c5a8a0f88f1c0fb30bce47219bcbbdcd9ef40a74d732fR51